src/Scripts/Game/Core/EL_Utils.c

```diff
@@ -48,6 +48,18 @@ class EL_Utils
 		return SCR_BaseContainerTools.GetPrefabResourceName(entity.GetPrefabData().GetPrefab());
 	}
 
+	//------------------------------------------------------------------------------------------------
+	static RplId GetRplId(IEntity entity)
```

Comment:
Already exists in Replication::FindId

Reply:
I think using the FindId actually does a full lookup for the entity over the already cached information inside the RplComponent, hence I made a convenience method to access the info from there.